server/@types/educationAndWorkPlanApi/index.d.ts

```diff
@@ -1821,13 +1993,19 @@ export interface operations {
         page?: number
         pageSize?: number
       }
+      header?: never
       path: {
         prisonNumber: string
       }
+      cookie?: never
     }
+    requestBody?: never
     responses: {
       /** @description OK */
       200: {
+        headers: {
+          [name: string]: unknown
+        }
```

Comment:
i assume all this is related to some renovate/dependabot dependency update?

Reply:
Yeah; I think there has been a recent update to `openapi-typescript` which we use to generate these files from the swagger spec; which has resulted in this extra "stuff" 🤷‍♂️